deploy/kubernetes/csi-attacher-nfsplugin.yaml

```diff
@@ -16,12 +16,15 @@ spec:
 
 ---
 kind: StatefulSet
-apiVersion: apps/v1beta1
+apiVersion: apps/v1
 metadata:
   name: csi-attacher-nfsplugin
```

Comment:
We actually don't need attacher anymore starting in 1.14. We can set CSIDriver.attachRequired = false.

Reply:
perfect, thanks, removed the attacher and added CSIDriver in https://github.com/kubernetes-csi/csi-driver-nfs/pull/24/commits/50d5f44d9c49bb118fc5a784d1f880fdddae02a4

and starting to have a tiny idea how does CSI work, finally :)

Reply:
Thanks, msau42!
With the latest commit this still checks out as I use it from manila-csi ...